evals/swebench/smoke-test.sh

```
@@ -0,0 +1,142 @@
+    print(f"ERROR: instance {target} not found", file=sys.stderr)
+    sys.exit(1)
+PY
+
+SCRATCH="$(mktemp -d -t claude-mem-smoke.XXXXXX)"
+trap 'rm -f "$CREDS_FILE" "$INSTANCE_JSON"; rm -rf "$SCRATCH"' EXIT
+
+# Parse the instance JSON once: print repo + base_commit to stdout, write the
+# problem statement directly to $SCRATCH/problem.txt. Avoids three separate
+# python3 invocations and any shell-quote injection on $INSTANCE_JSON.
+read -r REPO BASE_COMMIT < <(
+  python3 - "$SCRATCH" < "$INSTANCE_JSON" <<'PY'
```

Shellcheck (0.11.0)

[error] Multiple redirections compete for stdin. Use cat, tee, or pass filenames instead.

(SC2261)